YaST/Repos/_openSUSE_Leap_42.1_Default.xml.in

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0"?>
+<!-- Main Repositories for openSUSE Leap 42.1 
+     These are maintained by Stephan Kulow <coolo@suse.de> and Dominique Leuenberger <dimstar@suse.de> -->
+<metapackage xmlns:os="http://opensuse.org/Standards/One_Click_Install" xmlns="http://opensuse.org/Standards/One_Click_Install">
+    <group distversion="openSUSE Main Repository">
+	<repositories>
+
+	    <repository recommended="true" format="yast">
+		<_name>Main Repository (OSS)</_name>
+		<_summary>Main repository of openSUSE Leap 42.1 including only
+		Open Source Software</_summary>
+		<_description>The big Open Source Software (OSS) repository
+		for openSUSE Leap 42.1, giving you access to thousands of
+		packages maintained by the openSUSE team. (Due to the size of
+		the repository, adding it may take some time. 512 MB of RAM
+		or an enabled SWAP partition are highly recommended in case
+		you want to access this repository at installation time.)
+		 </_description>
+		<url>http://download.opensuse.org/distribution/leap/42.1/repo/oss/</url>
+	    </repository>
+	    <repository recommended="true" format="yast">
+		<_name>Main Repository (NON-OSS)</_name>
+		<_summary>Non-Open Source Software Addon repository for openSUSE Leap 42.1</_summary>
+		<_description>The official openSUSE Leap 42.1 repository for all
+		Non-Open Source Software maintained by the openSUSE team,
+		including Opera, Java, Flash, and more.</_description>
+		<url>http://download.opensuse.org/distribution/leap/42.1/repo/non-oss/</url>
```

Comment:
Removed Java, mentioned Steam in place (which is something users might be trying to find after all and we have it in the repo)